accounts/abi/error.go

```diff
@@ -84,10 +84,10 @@ func (e Error) String() string {
 
 func (e *Error) Unpack(data []byte) (interface{}, error) {
 	if len(data) < 4 {
-		return "", errors.New("invalid data for unpacking")
+		return "", errors.New("invalid data for unpacking, length is less than 4")
```

Comment:
Maybe something like this?
```suggestion
		return "", fmt.Errorf("invalid data length for unpacking: got %v, want 4", len(data))
```